string shape
string color
float64 x
float64 y
